manipvortex/manipulator_test.go

```diff
@@ -97,6 +97,7 @@ func Test_New(t *testing.T) {
 			newIdentityProcessor(manipulate.ReadConsistencyDefault, manipulate.WriteConsistencyDefault),
 			testmodel.Manager(),
 			OptionUpstreamManipulator(m),
+			OptionPrefetcher(NewDefaultPrefetcher()),
```

Comment:
I am not sure why we are changing the default behavior here.

Reply:
Prefetcher should be optional. We don't need a default for it. Otherwise the caller has to have an option `OptionPrefetcher(nil)` to disable it, which doesn't make sense.